### ISSUE TITLE
Kafka Option

### ISSUE THREAD

**Issue by ytooyama** [feldera/feldera#3240]
**Describe the bug**
If you don't want to use Redpanda for licensing reasons, it would be good to have the option to use Apache kafka.
It is best if you can try an environment that evaluates with docker compose.

**Comment by mihaibudiu** [comment]
I don't think we require RedPanda, in principle everything we do should work with Kafka.
What exactly you think would not work?

**Comment by ytooyama** [comment]
https://github.com/feldera/feldera/blob/main/deploy/docker-compose.yml#L45

Redpanda is a BSL license. Now you may be able to get the source for free and the software for free, but there is no guarantee that this will be the case in the future.
It is best to have the option to use components that explicitly have an OSS license.

**Comment by mihaibudiu** [comment]
I have changed the title of your issue. Please edit again if it does not reflect what the actual issue is.
It should be easy to do this, but because of the holidays I don't know how quickly this will happen - some people are on vacation.

**Comment by ytooyama** [comment]
Oh, sorry.
Please enjoy your vacation. I'm not in a hurry.

**Comment by snkas** [comment]
Thanks for the issue! Yes, using Kafka instead of Redpanda also works. Beyond updating the docker compose file it also needs some edits in the demos (`project_demo0{1, 3, 4, 6, 8}-...`) to replace their usage of `rpk` in Python. I've assigned the issue to myself.
